### PR TITLE
Add new Creating Schemas procedure

### PR DESCRIPTION
**Pull request by chenette**
Also add a schema blurb-and-link in Running Grid on Splinter and a link in the doc's left-nav TOC.

Signed-off-by: Anne Chenette <chenette@bitwise.io>

### CODE REVIEW

File: docs/0.1/creating_schemas.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+
+1. Set `GRID_DAEMON_ENDPOINT` to the Grid daemon's endpoint (URL and port),
+   such as `http://localhost:8080`. This environment variable replaces the
+   `-url` option on the `grid` command line.
+
+   ```
+   root@gridd-alpha:/# export GRID_DAEMON_ENDPOINT="http://localhost:8080"
+   ```
+
+1. Set `GRID_SERVICE_ID` to the fully qualified service ID (such as
```
Comment (agunde406) [inline docs/0.1/creating_schemas.md:120]: Can you add "For Grid on Splinter: " to the start of this test like it is in creating organization.

**Review by agunde406 — CHANGES_REQUESTED** [review]

File: docs/0.1/creating_schemas.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+* Existing public/private key files in `$HOME/.grid/keys/` (as generated by
+  `grid keygen`). The example environment uses the key files
+  `$HOME/.grid/keys/alpha-agent.priv` and `$HOME/.grid/keys/alpha-agent.pub` to
+  indicate that you will be acting as your organization's agent to add schemas
+  from the alpha node.
+
+## Procedure
+
+**IMPORTANT**: The commands in this procedure show host names, IDs, Docker
+container names, and other values from the example Grid-on-Splinter environment
```
Comment (agunde406) [inline docs/0.1/creating_schemas.md:70]: Grid-on-Splinter -> Grid on Splinter (no hyphens)

**Review by agunde406 — CHANGES_REQUESTED** [review]

File: docs/0.1/creating_schemas.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+   * Set of item properties (each with a name, data type, and value
+     that conforms to the data type)
+     <br><br>
+
+   The following example shows how to use `cat` to create the file
+   `product_schema.yaml`, with sample contents that match the product definition
+   shown in [Creating Products]({% link docs/0.1/creating_products.md %}).
+
+   ```
+   root@gridd-alpha:/# cat > product_schema.yaml
```
Comment (dnewh) [inline docs/0.1/creating_schemas.md:151]: Using `cat` to create these YAML files has caused issues in the past with incorrect formatting. I would suggest rewriting these steps to use `docker cp` similar to the grid on splinter example in the Grid repository.

**Review by dnewh — CHANGES_REQUESTED** [review]

**Review by dnewh — APPROVED** [review]

File: docs/0.1/creating_schemas.md (hunk 3 above)
Comment (chenette) [reply docs/0.1/creating_schemas.md:151]: Thanks!  That's good to know.  For time reasons, this will need to be done in a separate story.

**Pull request merged**